AK/BigIntBase.h

```diff
@@ -580,6 +594,72 @@ struct StorageOperations {
         if (size2 < size && (sign1 ^ sign2))
             negate(result, result);
     }
+
+    static constexpr void div_mod_internal(
+        StorageSpan<WordType, false> dividend, StorageSpan<WordType, false> divisor,
+        StorageSpan<WordType, false> quotient, StorageSpan<WordType, false> remainder,
+        size_t dividend_len, size_t divisor_len)
+    {
+        // Knuth's algorithm D
+        // D1. Normalize
+        // FIXME: Investigate GCC producing bogus -Warray-bounds when dividing u128 by u32. This code
```

Comment:
Is this still an issue after a move?

Reply:
Not 100% sure, but would not trust whatever the GCC toolchain will tell me, as we allow quite a few  GCC versions, and it could be something they fixed somewhere down the line

Reply:
it most likely is, -Warray-bounds fires on all sorts of nonsense with ASAN and UBSAN enabled.